README.md

```diff
@@ -1,2 +1,105 @@
 # SeineSailor
-Help developers navigating through the codebase smoothly
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![GitHub](https://img.shields.io/github/last-commit/SeineAI/SeineSailor/main?style=flat-square)](https://github.com/SeineAI/SeineSailor/commits/main)
+
+## Overview
+
+SeineSailor is an AI-powered copilot for project managers, designed to assist with code reviews, project discussions,
+and knowledge transfer. It leverages the power of WatsonX's `llama3` and `mixt-8x7B` models to provide intelligent and
+context-aware support throughout the development lifecycle.
+
+## Features
+
+- **PR Review and Summarization**: SeineSailor can be summoned to review pull requests and provide a summary of the
+  changes when mentioned in a PR comment.
+- **Project Discussion Participation**: SeineSailor can actively participate in project discussions and brainstorming
+  sessions when invited using the `@SeineSailor` mention in issues or other discussion forums.
+- **Knowledge Transfer**: SeineSailor maintains a knowledge base of the project's development history and can assist new
+  developers by answering questions about the codebase, architecture, and function implementations.
+- **Customizable Prompts**: Tailor the `system_message`, `summarize`, and other prompts to focus on specific aspects of
+  the review process or change the review objective.
+
+To use SeineSailor, you need to add the provided YAML file to your repository and configure the required environment
+variables, such as `GITHUB_TOKEN` and `WATSONX_API_KEY`. For more information on usage, examples, and contributing,
+refer to the sections below.
+
+## Installation
```

Comment:
@SeineSailor Add single comment

[consolidate test_env into llm-review](https://github.com/SeineAI/SeineSailor/commit/dc145896169a08fbc983e2a66d4e6c6b2a4fb412)

Reply:
@SeineSailor issue_comments will be skipped,

let's test PR comments.

Reply:
@SeineSailor enabled debug logger. let's see what are the values